### ISSUE TITLE
bad link in readme

### ISSUE THREAD

**Issue by JulianNorton** [Elucidation/tensorflow_chessbot#16]
http://tetration.xyz/tensorflow_chessbot/

> Looks good. Now that we've loaded the data, let's build up a simple softmax regression classifier based off of **this beginner tutorial on tensorflow.**

dead link:
https://www.tensorflow.org/versions/v0.6.0/tutorials/mnist/beginners/index.html
